frontend/src/pages/project/components/ProjectForm.vue

```diff
@@ -157,6 +158,7 @@
                 v-model:confirmed="input.billingConfirmation"
                 :project-type="selectedProjectType"
                 :subscription="subscription"
+                :trialMode="team.billing?.trial && (!team.billing?.active || (team.billing.trialProjectAllowed && selectedProjectType?.id === this.settings['user:team:trial-mode:projectType']))"
```

Comment:
I'd extract this into a computed prop to add a bit more explanation with what's going on

Reply:
done